BeforeBuild.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
General question/comment: in addition to the unit tests and this port successfully building, what other testing has been performed to verify everything passes sanity?

Reply:
Mostly Unit Tests passing and I've tested installing the SARIF Viewer and opening SARIF files within it.  I've also converted a few files to SARIF with the converter but this was before it was replaced with Multitool.

Reply:
Mostly Unit Tests passing and I've tested installing the SARIF Viewer and opening SARIF files within it.  I've also converted a few files to SARIF with the converter but this was before it was replaced with Multitool.

---
In reply to: [141489911](https://github.com/Microsoft/sarif-sdk/pull/624#discussion_r141489911) [](ancestors = 141489911)